runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -26,26 +26,35 @@
 
 import com.google.common.collect.ImmutableSet;
 import javax.inject.Inject;
+
+import com.google.inject.Provides;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
+import net.runelite.api.*;
```

Comment:
Remove the wildcard import. You'll also want to tell your IDE to not make them wildcards unless they're static imports

Reply:
So unless there's only 1 class in that package, always import with *? Wouldn't the config imports need to be imported with * too?

Reply:
Even if there's only 1 class in the package, it should **not** be imported with wildcards **ever**. The exception is excessive static imports (such as ItemIDs)